tests/commands/changed_languages.py

```diff
@@ -34,10 +35,15 @@ def test_changed_languages_since_revision(capfd, afrikaans_tutorial,
                                           french_tutorial):
     """Changed languages since a given revision"""
     # Everything
-    call_command('changed_languages', '--after-revision=0')
+    rev = afrikaans_tutorial.stores.aggregate(
+        rev=Min('last_sync_revision'))['rev'] - 1
```

Comment:
Shouldn't it be the minimum revision for both `afrikaans_tutorial` and `french_tutorial`?